src/net/larse/lcms/algorithms/VCT.java

```diff
@@ -47,12 +51,8 @@
  */
 public class VCT {
 
-//  public final static class VCTSolver {
```

Comment:
Matt's question is whether we need to use the inner class. Currently the
thought is to have VCT instantiated and the objects/arrays within it will
be reused.


On Wed, May 27, 2015 at 12:05 PM, gorelick <notifications@github.com> wrote:

> In src/net/larse/lcms/algorithms/VCT.java
> <https://github.com/c11/lcms/pull/5#discussion_r31168772>:
>
> > @@ -47,12 +51,8 @@
> >   */
> >  public class VCT {
> >
> > -//  public final static class VCTSolver {
>
> Whether you call it a solver or not, you need a class that gets
> instantiated and initialized once, that you can reuse over and over, so you
> don't end up recreating zillions of objects/arrays.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/c11/lcms/pull/5/files#r31168772>.
>